.travis.yml

```diff
@@ -43,8 +51,14 @@ install:
 
 script:
   - set -e
-  - py.test -s -rxs -v
-  - conda install -n root conda-build anaconda-client
-  - conda build conda-recipe --python $TRAVIS_PYTHON_VERSION
-  - conda install pocean-core --use-local
-  - ./docs/deploy.sh
+
+  - if [[ $TEST_TARGET == 'default' ]]; then
+      py.test -s -rxs -v ;
+    fi
+
+  - if [[ $TEST_TARGET == "docs" ]]; then
+      conda install -n root conda-build anaconda-client ;
+      conda build conda-recipe --python $TRAVIS_PYTHON_VERSION ;
+      conda install pocean-core --use-local ;
```

Comment:
Any special reason to use a conda recipe for this? I guess that a pip install inside the test env would suffice.